docs/operator_parameters.md

```diff
@@ -0,0 +1,175 @@
+  name, '*{team}*' denotes the team name and '*{hostedzone}*' is the
+  **db_hosted_zone** value.
+ 
+### AWS or GSC interaction
+* **wal_s3_bucket**
+  S3 bucket to use for shipping WAL segments with WAL-E. A bucket has to be
+  present and accessible by Patroni managed pods. At the moment, supported
+  services by Spilo are S3 and GCS.
+
+* **log_s3_bucket**
+  S3 bucket to use for shipping Postgres daily logs. Works only with S3 on AWS.
+  The bucket has to be present and accessible by Patroni managed pods. At the
+  moment Spilo does not yet support this.
+
+* **kube_iam_role**
+  AWS IAM role to supply in the *iam.amazonaws.com/role* annotation of Patroni
+  pods. Only used when combined with
+  [kube2iam](https://github.com/jtblin/kube2iam) project on AWS
+
+### Debugging the operator
+* **debug_logging**
+  boolean parameter that switches verbose debug logs from the operator. Enabled
+  by default.
+
+* **enable_db_access**
+  boolean parameter that toggles the functionality of the operator that require
+  access to the Postgres database, i.e. creating databases and users. Enabled
+  by default.
+
+### Automatic creation of database human users
+* **enable_teams_api**
+  boolean parameter that toggles usage of the Teams API by the operator.
+  Enabled by default.
+
+* **teams_api_url**
+  contains the URL of the Teams API service. There is a [demo
+  implementation](https://github.com/ikitiki/fake-teams-api). Default is
+  https://teams.example.com/api/. 
+
+* **team_api_role_configuration**
+  Postgres parameters to apply to each team member role. Default is
+  '*log_statement:all*'. It is possible to supply multiple options, separating
+  them by commas. Options containing commas within the value are not supported,
+  with an exception of the '*search_path*'. For instance 
+
+  ```yaml
+  teams_api_role_configuration: "log_statement:all,search_path:'data,public'"
+  ```
+
+* **enable_team_superuser**
+  whether to grant superuser to team members created from the Teams API.
+  Disabled by default.
+
+* **team_admin_role**
+  role name to grant to team members created from the Teams API. Default is
+  '*admin*', which is the role created by Spilo.
+
+* **pam_role_name**
+  when set, the operator will add all team member roles to this group and add
+  a *pg_hba* line to authenticate members of that role via pam. Default is '*zalandos*'.
+
+* **pam_configuration**
+  when set, should contain a URL to use for authentication against the user
+  name and the token supplied as the password.  Used in conjunction with
+  [pam_oauth2](https://github.com/CyberDem0n/pam-oauth2) module. Default is
+  "*https://info.example.com/oauth2/tokeninfo?access_token= uid
+  realm=/employees*"
+
+### Logging and REST API
+* **api_port**
+  REST API listener listens to this port. Default is *8080*.
+
+* **ring_log_lines**
+  number of lines in the ring buffer used to store cluster logs. Default is *100*.
+
+* **cluster_history_entries**
```

Comment:
and the purpose (and difference between) `ring_log_lines` and `cluster_history_entries` is ?